src/services/YotiService.ts

```diff
@@ -252,8 +266,14 @@ export class YotiService {
   		method: HttpVerbsEnum.GET,
   		endpoint: `/sessions/${sessionId}/instructions/pdf`,
   	});
+	  const yotiRequestConfig =  yotiRequest.config!;
```

Comment:
Can this be moved into generateYotiRequest method to accept these two config params and set if they are present pls